packages/space-opera/src/components/mobile_view/open_mobile_view.ts

```diff
@@ -207,12 +213,17 @@ export class OpenMobileView extends ConnectedLitElement {
   }
 
   async sendSessionContentHolder(
-      session: MobileSession, updatedContent: EditorUpdates,
+      session: MobileSession, updatedContent: EditorUpdates, posterBlob: Blob,
       usdzBlob: Blob|undefined, gltfBlob: Blob|undefined,
       envBlob: Blob|undefined) {
     try {
       await this.sendSessionContent(
-          session, {...updatedContent}, usdzBlob, gltfBlob, envBlob);
+          session,
```

Comment:
Looks like editor state is serialized and restored? Is that right?

Reply:
Yeah, in order to deploy to your mobile device, it  has to serialize everything and post it through the  piping-server for the mobile to fetch. It doesn't serialize all of the Redux state, since the mobile doesn't care; just enough to recreate the MV element.